scripts/lib-franklin.js

```diff
@@ -172,20 +175,34 @@ export async function fetchPlaceholders(prefix = 'default') {
         })
         .then((json) => {
           const placeholders = {};
-          json.data.forEach((placeholder) => {
-            placeholders[toCamelCase(placeholder.Key)] = placeholder.Text;
+          const KEY = 'Key';
+
+          json.data.forEach((entry) => {
+            Object.keys(entry).forEach((localeKey) => {
+              if (localeKey !== KEY) {
+                if (placeholders[localeKey]) {
+                  placeholders[localeKey][entry.Key.toLowerCase()] = entry[localeKey];
```

Comment:
Should this be `toCamelCase` as was being done earlier instead of `toLowerCase` ?

Reply:
toCamelCase was there, but it was not working as i think there is some problem in toCamelCase function.
And also for a key i thought lowercase would be good rather than camel case. WDYT ?

Reply:
ok. I think if we are changing this, we should check all the placeholders in code and ensure they are all in lower case. [Looking here](https://github.com/search?q=repo%3Ahlxsites%2Fzeiss%20placeholders&type=code), that seems to be the case already.